README.md

```diff
@@ -1 +1,43 @@
-# Ten99Policy Node.js Library
+# 1099Policy Node.js Library
+
+Node.js client library for 1099Policy platform.
+
+## Requirements
+
+Node 8, 10 or higher.
+
+## Installation
+
+Install the package with:
+
+```sh
+soon
+```
+
+## Usage
```

Comment:
We can use this language from the API docs for this section:

"To use the 1099Policy API you need to authenticate requests using API keys. Sign up for a developer account to view and manage your API keys from the the 1099Policy Dashboard. https://dashboard.1099policy.com/register"